src/include/daos/object.h

```diff
@@ -87,8 +87,8 @@ daos_ec_cs_valid(uint32_t cell_sz)
 	if (cell_sz < DAOS_EC_CELL_MIN || cell_sz > DAOS_EC_CELL_MAX)
 		return false;
 
-	/* should be multiplier of the min size */
-	if (cell_sz % DAOS_EC_CELL_MIN != 0)
+	/* should be multiplier of the min size, EC/ISAL lib require 32 byte alignment */
```

Comment:
"multiplier of the min size" the comment need change a little bit.
I'm not sure if really need to set cell size like that, but looks not bad to release the limitation.

BTW, seems ISAL does not require 32B align, just >=32B.

Reply:
>I'm not sure if really need to set cell size like that, but looks not bad to release the limitation.

Oh, this is for IOR hard test(-t 47008) and similar application,  by setting ec_cell_sz:47008, I can see read performance improvement (1400MB to 1800MB) in my test.

> BTW, seems ISAL does not require 32B align, just >=32B.

hmm, not sure what does alignment >= 32B mean? can we define the alignment for ISAL?

Reply:
> > BTW, seems ISAL does not require 32B align, just >=32B.
> 
> hmm, not sure what does alignment >= 32B mean? can we define the alignment for ISAL?

I mean for ISAL API, seems the len only need to >= 32B, not must be multiplier of 32B.